### PR TITLE
[DEMO-SMF-UPF-INTEG]: Temp commit SMF-UPF INTEG

### PR DESCRIPTION
**Pull request by panyogesh**
1. Integrated UPF patch
2. Mereged with SMF-AMF stubs to test
   session creation/deletion

Test:
1. Create Session
2. Delete Session

Signed-off-by: YOGESH PANDEY <yogesh@wavelabs.ai>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by panyogesh** [comment]
[UT_logs_3806.txt](https://github.com/magma/magma/files/5652241/UT_logs_3806.txt)

**Comment by themarwhal** [comment]
@panyogesh hello! is this PR still relevant? If it is not, please close it!

File: lte/gateway/c/session_manager/SessionStateEnforcer.cpp [hunk 1]
```diff
@@ -487,7 +487,6 @@ bool SessionStateEnforcer::static_rule_init() {
   reqpdr1.mutable_pdi()->set_net_instance("uplink");
   reqpdr1.set_o_h_remo_desc(0);
   reqpdr1.mutable_set_gr_far()->add_far_action_to_apply(Value);
-  reqpdr1.mutable_set_gr_far()->set_far_id(count);
   reqpdr1.mutable_activate_flow_req()->mutable_request_origin()->set_type(
```
Comment (uri200) [inline lte/gateway/c/session_manager/SessionStateEnforcer.cpp:490]: are all those fileds supposed to go away with this PR?

File: lte/gateway/configs/pipelined.yml [hunk 2]
```diff
@@ -162,8 +162,13 @@ ovs_gtp_stats_polling_interval: 180
 he_proxy_eth_mac: 'e6:8f:a2:80:80:80'
 he_enabled: true
 
+# Paging timeout value
+paging_timeout: 30
+
 # Make as a True when gnb ip will support for uplink 
 ovs_multi_tunnel: False
+# Paging timeout value
+paging_timeout: 30
```
Comment (uri200) [inline lte/gateway/configs/pipelined.yml:171]: i see this twice. This may mean this PR was not merged properly and other issues may have been introduced

Comment (panyogesh) [reply lte/gateway/configs/pipelined.yml:171]: Will close the PR. Meant only to provide integrated UPF code for SMF to test (for the demo). As the UPF patches are individually landed this PR is not needed.

**Comment by panyogesh** [comment]
Individual PRs landed so closing this PR.

**Pull request closed without merging**